bot.py

```diff
@@ -255,23 +284,33 @@ async def _snippets(self, ctx, *args):
 
         doc = db.find_one({'_id': args[0]})
 
-    @commands.has_any_role(config.modRole)
-    @commands.group(name='appeal', case_insensitive=True, invoke_without_command=True)
-    async def _appeal(self, ctx):
-        return await ctx.send_help(self._appeal)
-
-    @commands.has_any_role(config.modRole)
-    @_appeal.command(name='accept')
-    async def _appeal_accept(self, ctx, *, reason):
+    @cog_ext.cog_subcommand(
+        base='appeal',
+        name='accept',
+        guild_ids=guildList,
+        base_description='Make a decision to accept or deny a ban appeal',
+        description='Accept a user\'s ban appeal',
+        base_permissions={config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]},
```

Comment:
`{config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]}` should be made into a static variable and passed instead.